incubator/jaeger/templates/collector-deploy.yaml

```diff
@@ -31,9 +32,64 @@ spec:
       nodeSelector:
 {{ toYaml .Values.collector.nodeSelector | indent 8 }}
       containers:
-      - name: "{{ template "jaeger.fullname" . }}-collector"
-        image: "{{ .Values.collector.image }}:{{ .Values.collector.tag }}"
+      - name: {{ template "jaeger.fullname" . }}-collector
+        image: {{ .Values.collector.image }}:{{ .Values.collector.tag }}
         imagePullPolicy: {{ .Values.collector.pullPolicy }}
+        env:
+          - name: SPAN_STORAGE_TYPE
```

Comment:
I think this should be `SPAN_STORAGE` but I could not confirm that during my testing..
https://github.com/jaegertracing/jaeger/blob/master/plugin/storage/factory_config.go#L24-L30

Reply:
Run the `hotrod` app to check that it works fine. They are both accepted. If you run `docker run -it --rm jaegertracing/jaeger-collector /go/bin/collector-linux -h`, you'll see that the collector accepts the following flag: `--span-storage.type`. This also automatically means that it would get the value from the uppercased variable with the same name where `-` and `.` are replaced with `_` (i.e. `SPAN_STORAGE_type`). This is mentioned in [the documentation](http://jaeger.readthedocs.io/en/latest/deployment/#configuration).

Reply:
NB: https://github.com/jaegertracing/jaeger/issues/642

Reply:
I merged https://github.com/jaegertracing/jaeger/issues/642, the env var is back to `SPAN_STORAGE_TYPE` (will be released in 1.2)